docs/reference/requirement-sets/add-in-commands-requirement-sets.md

```diff
@@ -28,6 +28,17 @@ The following table lists the add-in commands 1.1 requirement set, the Office cl
 
 >\* The [Office.context.requirements.isSetSupported](/javascript/api/office/office.requirementsetsupport#issetsupported-name--minversion-) method will erroneously return `false` for versions 16.9 &ndash; 16.14 (inclusive), but the requirement set *is* supported on these versions.
 
+The add-in commands 1.3 requirement set introduces manifest markup that enables an add-in to customize the placement of a custom tab on the Office ribbon and to insert built-in Office ribbon controls into custom control groups.
```

Comment:
Curious about what happened to 1.2?

Reply:
That's going to include the contextual tabs feature which might actually go to public preview after 1.3.